phones/models.py

```diff
@@ -291,9 +292,9 @@ def save(self, *args, **kwargs):
         # as realphone
         self.country_code = realphone.country_code.upper()
 
-        # Add US numbers to the Relay messaging service, so it goes into our
-        # US A2P 10DLC campaign
-        if use_twilio and self.country_code == "US":
+        # Add numbers to the Relay messaging service, so it goes into our
+        # US A2P 10DLC campaigns
```

Comment:
quibble/question: Does this comment also need to have "US" removed from it? Not sure if A2P 10DLC campaigns are country-specific?
```suggestion
        # A2P 10DLC campaigns
```

Reply:
Yep, Twilio just calls them A2P 10DLC - https://www.twilio.com/en-us/a2p-10dlc